src/HotChocolate/Spatial/test/Types.Tests/GeoJsonPointSerializerTests.cs

```diff
@@ -198,7 +198,7 @@ public void ParseLiteral_Should_Pass_When_CorrectGeometry(string typeName)
     }
 
     [Theory]
-    [InlineData(PointInputName)]
+    [InlineData(CoordinatesTypeName)]
```

Comment:
I am not sure if this is correct. 
I mean the whole point of these tests is  to test the GeoJsonPointInputType

This these follows the same priciple as e.g. the Polygon one: 

https://github.com/ChilliCream/hotchocolate/blob/2413375816427dd79c9ba7c7d3e3cf8c6ec77bc3/src/HotChocolate/Spatial/test/Types.Tests/GeoJsonPolygonSerializerTests.cs#L257-L274

So i guess the PointInputTYpe does not throw an exception, though it actually should throw one

Reply:
@michaelstaib  made a change to the input parser and it comes up with a default type and because an input type has a type int he schema it generates a default point type

Reply:
https://github.com/ChilliCream/hotchocolate/commit/0b98f7879364b2f71d7303030828a98e501e526a

Reply:
Sorry for the slow response time on this ... pascal is doing is military service and I am trying to keep with all up on my own at the moment :) 

Yes, I did rework the whole process on the parser. The question is if the new behavior is OK. If so we can remove this test case.... But I have no clue... not my topic. @PascalSenn will need to have another look when he has some time.

Reply:
@PascalSenn anymore thoughts on this?